src/box/applier.cc

```diff
@@ -1486,7 +2221,10 @@ applier_subscribe(struct applier *applier)
 	/*
 	 * Process a stream of rows from the binary log.
 	 */
-	applier_subscribe_f(applier);
+	if (replication_num_applier_threads > 0)
+		applier_thread_subscribe(applier);
+	else
+		applier_subscribe_f(applier);
```

Comment:
Why do you want to keep the old applier-in-fiber implementation? Applier-in-thread does the same job and it's supposed to be faster so I don't think we need to keep the legacy implementation around.

Reply:
Dropped that